### PR TITLE
Some translation fixes

### PR DESCRIPTION
**Pull request by SomeTroglodyte**
#6438 redo

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/Difficulty.kt [hunk 1]
```diff
@@ -47,7 +47,7 @@ class Difficulty: INamed, ICivilopediaText {
     override fun getCivilopediaTextLines(ruleset: Ruleset): List<FormattedLine> {
         val lines = ArrayList<FormattedLine>()
         lines += FormattedLine("Player settings", header = 3)
-        lines += FormattedLine("{Base Happiness}: $baseHappiness ${Fonts.happiness}", indent = 1)
+        lines += FormattedLine("{Base happiness}: $baseHappiness ${Fonts.happiness}", indent = 1)
```
Comment (xlenstra) [inline core/src/com/unciv/models/ruleset/Difficulty.kt:50]: Wasn't the plan that after #5963 and 8311e65120c4dec86ecd51fda1e9dc6a67f0c6e2 we were going to capitalize all these game terms?

Comment (SomeTroglodyte) [reply core/src/com/unciv/models/ruleset/Difficulty.kt:50]: I didn't care much whether 'Base happiness' or 'Base Happiness' stays in, but cared enough to not leave _both_ in the project (one was untranslatable which is why I found it). So I threw some dice skewing for what I saw elsewhere around the project.

> modify this accordingly

Kudos - making your own decisions is always worthy some respect :grin:

**Comment by SomeTroglodyte** [comment]
Oh, and 
```html
<details><summary>Example</summary>

<img src="https://user-images.githubusercontent.com/63475501/140308518-ad5a2f50-d5f1-4467-a296-3a67f6d0b007.png" />

</details>
```
works

**Comment by HaneulCheong** [comment]
> Oh, and
> 
> ```
> <details><summary>Example</summary>
> 
> 
> 
> </details>
> ```
> 
> works

Sorry to inturrupt, but...is this for #6453?

**Pull request merged**